### PR TITLE
AGG_SIG_ME and AGG_SIG_UNSAFE take exactly two arguments

### PR DESCRIPTION
**Pull request by arvidn**
additional ones are errors

### CODE REVIEW

File: src/gen/conditions.rs [hunk 1]
```diff
@@ -1577,7 +1611,7 @@ fn test_agg_sig_unsafe_invalid_pubkey() {
 fn test_agg_sig_unsafe_invalid_msg() {
     // AGG_SIG_ME
     assert_eq!(
-        cond_test("((({h1} ({h2} (123 (((49 ({pubkey} ({longmsg} )))))")
+        cond_test("((({h1} ({h2} (123 (((50 ({pubkey} ({longmsg} )))))")
```
Comment (arvidn) [inline src/gen/conditions.rs:1614]: this was a copy-paste error from before. `AGG_SIG_UNSAFE` is 49 and `AGG_SIG_ME` is 50

File: src/gen/conditions.rs [hunk 2]
```diff
@@ -1575,7 +1609,7 @@ fn test_agg_sig_unsafe_invalid_pubkey() {
 
 #[test]
 fn test_agg_sig_unsafe_invalid_msg() {
-    // AGG_SIG_ME
+    // AGG_SIG_UNSAFE
```
Comment (arvidn) [inline src/gen/conditions.rs:1612]: copy-paste typo from before

File: src/gen/conditions.rs [hunk 3]
```diff
@@ -93,13 +93,21 @@ fn parse_args(
...
+                SExp::Pair(_, _) => Err(ValidationErr(c, ErrorCode::InvalidCondition)),
+                _ => Ok(Condition::AggSigUnsafe(pubkey, message)),
+            }
         }
         AGG_SIG_ME => {
             let pubkey = sanitize_hash(a, first(a, c)?, 48, ErrorCode::InvalidPubkey)?;
             c = rest(a, c)?;
             let message = sanitize_announce_msg(a, first(a, c)?, ErrorCode::InvalidMessage)?;
-            Ok(Condition::AggSigMe(pubkey, message))
+            // AGG_SIGG_ME takes exactly two parameters
```
Comment (richardkiss) [inline src/gen/conditions.rs:106]:
```suggestion
            // AGG_SIG_ME takes exactly two parameters
```

**Review by richardkiss — APPROVED** [review]
Looks good.

**Pull request merged**